### ISSUE TITLE
Lacking properties (?)

### ISSUE THREAD

**Issue by Podlas29** [slamdata/purescript-css#19]
Hi. I wanted to use this module, but I found that it lacks many properties and/or values. For example there is no float and flex - related properties and values. Is there some reason not to include them, or module is just incomplete, and merge requests with more definitions will be accepted?

**Comment by garyb** [comment]
Yeah, it's just incomplete, contributions would definitely be welcome!

**Comment by Podlas29** [comment]
If someone posts here some css properties/values that he would like to see in project, I will probably make an attempt to implement them, unless they are really obscure.
